tests/behat/features/roles_list.feature

```diff
@@ -0,0 +1,18 @@
+@tide
+Feature: Roles
+
+  Ensure that users's role are listed in the profile page
+
+  @api
+  Scenario Outline: User with assigned roles visits profile page
```

Comment:
hey @yeniatencio 
I think this may not be the correct test steps.
you should assign a user a role(s) and then check if you can see the text of the role's label(s).
in your test steps, `Then I should see the text "Roles"` , the `Roles` also exist in the tab. 
![CleanShot 2023-05-09 at 08 59 18](https://user-images.githubusercontent.com/8788145/236955275-36e685f7-b9c6-4a78-8dcc-a4050de37164.jpg)

Reply:
Hey @vincent-gao . I have updated behat test for your review.